src/amo/components/AddonDetail.js

```diff
@@ -90,6 +91,13 @@ class AddonDetail extends React.Component {
           <div dangerouslySetInnerHTML={sanitizeHTML(nl2br(addon.description),
                                                      allowedDescriptionTags)} />
         </section>
+
+        <hr />
```

Comment:
I used `hr` in the detail page as a quick fix, but since it's presentational markup we should reconsider keeping hrs around.